client/js/components/procedure/SegmentsList/FilterFlyout.vue

```diff
@@ -283,8 +299,16 @@ export default {
               operator: this.operator
             }
           }
+
+          if (this.memberOf !== '') {
```

Comment:
```suggestion
          if (this.memberOf) {
```
should be sufficient here

Reply:
Done in 07b5cfac90f11c0268ab448ecf24e98e080642b2